dataprofiler/profilers/unstructured_labeler_profile.py

```diff
@@ -88,6 +88,28 @@ def __add__(self, other):
 
         return merged_profile
 
+    def diff(self, other_profile, options=None):
+        cls = self.__class__
+        if not isinstance(other_profile, cls):
+            raise TypeError("Unsupported operand type(s) for diff: '{}' "
+                            "and '{}'".format(cls.__name__,
+                                              other_profile.__class__.__name__))
+
+        entity_counts_diff = {}
+        entity_percentages_diff = {}
+        for i in ['word_level', 'true_char_level', 'postprocess_char_level']:
```

Comment:
Kinda nitpicky, but I usually associate `i` with index, whereas these are different strings, so maybe use `key` or `level` or something to show that they are dictionary keys.